### ISSUE TITLE
All after mission salvage is valued a 0 c-bills

### ISSUE THREAD

**Issue by MaestroHimself** [MegaMek/mekhq#4812]
### Prerequisites and Pre-Issue Checklist

- [X] I'm reporting the issue to the correct repository:
- [MegaMek](https://github.com/MegaMek/megamek/issues)
- [MegaMekLab](https://github.com/MegaMek/megameklab/issues)
- [MekHQ](https://github.com/MegaMek/mekhq/issues)

- [X] I've tested the issue against at least the latest MILESTONE version
- [X] I've asked on the MegaMek Discord about the error
- [X] I've reviewed the BattleTech rules and MekHQ documentation, and I've confirmed that something isn't working as intended.
- [X] I've searched the Github tracker and haven't found the issue listed

### Severity *

Medium (Gameplay Limitation): Non-core functionality is impaired, providing a suboptimal but playable experience.

### Brief Description *

![0Salvage_1](https://github.com/user-attachments/assets/3f642a2b-c7d4-457c-b70f-e3d38dfd915e)
After completing a mission in Megamek, started from a StratOps Campaign in MegHQ, all salvage is listed at 0 c-bill value.
This happened on all 2 missions played on a new campaign so far, mission types were Convoy Attack and SupplyDeport - Hostile - Extract.

### Steps to Reproduce

1. Start new campaign on 0.50
2. Run company generator, accept contract, start mission
3. Play mission, win, own the field
4. Select that the field was owned, look at salvage, notice everything is valued at 0 c-bills

### Operating System *

Windows 10

### Java Version *

17.0.12

### MekHQ Suite Version *

v0.50.0

### Attach Files

[logs.zip](https://github.com/user-attachments/files/16943128/logs.zip)
[Campaign Settings and end of battle mul.zip](https://github.com/user-attachments/files/16943149/Campaign.Settings.and.end.of.battle.mul.zip)

### Final Checklist

- [X] I've checked to make sure that this issue has not already been filed
- [X] I'm reporting only one issue in this ticket for clarity and focus

**Comment by IllianiCBT** [comment]
Based on that screenshot nothing has been salvaged. What happens if you tick one of the salvage boxes?

**Comment by MaestroHimself** [comment]
No worries man. But just to be certain, if I tick a box to salvage something the value is still 0. Everything is and stays zero.

**Comment by IllianiCBT** [comment]
Can you go into campaign options and share a screenshot of the Finance tab?

**Comment by MaestroHimself** [comment]
![Finance_Tab](https://github.com/user-attachments/assets/49365482-8ee7-4246-bbfb-64fef580b47c)
Here you go. Looking at it, think it's the used parts modifers being 0.00?

**Comment by IllianiCBT** [comment]
Yeah, I'm going to assume your preset was made prior to 49.20? If so, you preserved a bug that existed in those versions where changing to 'reverse quality names' caused the Finance Tab to reset to default values. Which just happened to be 0.00 for parts value.

If you head onto the discord and check the mhq channel, there is a pinned comment that includes what the values should default to. Or, you can rebuild the preset in 50.0, whichever you prefer. There were a _lot_ of changes in 50.0 so that wouldn't be the worse idea.

**Comment by IllianiCBT** [comment]
Closing as resolved. If you believe this issue has been closed in error, please respond with your reasoning and we can get it re-opened.
